src/client/components/ListOfWeeksWithDetail/index.test.js

```diff
@@ -0,0 +1,25 @@
+import renderNonTransContent from "../../test-helpers/renderNonTransContent";
+import Component from "./index";
+
+describe("<ListOfWeeksWithDetail />", () => {
+  it("renders the component", async () => {
+    const weekOfAnswers = {
+      tooSick: false,
+      couldNotAcceptWork: false,
+      didYouLook: false,
+      refuseWork: false,
+      schoolOrTraining: false,
+      workOrEarn: false,
+    };
+
+    const wrapper = renderNonTransContent(Component, "ListOfWeeksWithDetail", {
+      userData: {
+        formData: weekOfAnswers,
+        weeksToCertify: [1, 2, 3],
+        programPlan: "UI full time",
```

Comment:
Hmm, does this need to be an array? The snapshot has weekProgramPlan equal to individual letters.

Reply:
good catch! funny, proptypes validates that it's an array of strings, but I guess in javascript a string qualifies as an array of strings? :p